### ISSUE TITLE
[Suggestion] Structure Finder

### ISSUE THREAD

**Issue by C0nfig1ssu3** [MeteorDevelopment/meteor-client#1369]
**Describe the feature**
Find Structures based on the seed or coordinates

**Screenshots**
this is a screenshot from Aristois client but not working well
![Immagine](https://user-images.githubusercontent.com/86430582/129453400-5de90cbc-1150-4f4f-8bdc-69e94f31b58c.png)

**Additional info**
for Stronghold Finder I suggest use the 2 ender eyes calculation and after show up on chat the coords for use #goto

**Comment by C10udburst** [comment]
Did you launch the client before making this suggestion? 🤡

There is `.locate` command and using seeds is coming in pr #1300

**Comment by C0nfig1ssu3** [comment]
> Did you launch the client before making this suggestion? 🤡
> 
> There is `.locate` command and using seeds is coming in pr #1300
> 
> Also this is a duplicate

yes and not works well for Nether fortess, did it in front of if ''no fortess found'' so frick off
